pkg/config/ebpf_tracer.go

```diff
@@ -48,4 +48,7 @@ type EBPFTracer struct {
 	// Enables the heuristic based detection of SQL requests. This can be used to detect
 	// talking to databases other than the ones we recognize in Beyla, like Postgres and MySQL
 	HeuristicSQLDetect bool `yaml:"heuristic_sql_detect" env:"BEYLA_HEURISTIC_SQL_DETECT"`
+
+	// Enables GPU instrumentation for CUDA kernel launches and allocations
+	InstrumentGPU bool `yaml:"instrument_gpu" env:"BEYLA_INSTRUMENT_GPU"`
```

Comment:
Just thinking out loud...

maybe some less generic term, in case we support other features or GPUs? Something like: `instrument_gpu_kernels`

Or maybe keep the name and replace it by a list type that could be expanded at some time:
```
instrument_gpu: ["kernel_calls", "gpu_allocations"]
```

Reply:
I was thinking maybe we add more in the future under that option. I think I'll mark it as experimental in the docs and I'll put a notice that the option may change in the future.